### PR TITLE
Introduce smoke tests for PCI Utils

### PR DESCRIPTION
**Pull request by rfan1**
- Related ticket: https://progress.opensuse.org/issues/165252
- Verification run: 
[opensuse](https://openqa.opensuse.org/tests/overview?distri=opensuse&build=rfan0927)
[sle](https://openqa.suse.de/tests/overview?build=rfan0927&distri=sle)

**Please ignore the failure on sle15sp7, it is https://bugzilla.suse.com/show_bug.cgi?id=1230995**

### CODE REVIEW

File: tests/console/pciutils.pm [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+use serial_terminal 'select_serial_terminal';
+use utils;
+use version_utils 'is_sle';
+use registration 'add_suseconnect_product';
+
+sub run {
+    select_serial_terminal;
+
+    add_suseconnect_product('sle-module-development-tools') if is_sle('>=15');
+    zypper_call('in cpupower powertop pciutils libpci3');
```
Comment (Vogtinator) [inline tests/console/pciutils.pm:23]: IMO it does not make sense to install libraries that way. The sover will change at some point.

IMO just use `pciutils` if you need a version number.

File: tests/console/pciutils.pm [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+use utils;
+use version_utils 'is_sle';
+use registration 'add_suseconnect_product';
+
+sub run {
+    select_serial_terminal;
+
+    add_suseconnect_product('sle-module-development-tools') if is_sle('>=15');
+    zypper_call('in cpupower powertop pciutils');
+    record_info('pciutis version:', script_output('rpm -q pciutils'));
```
Comment (foursixnine) [inline tests/console/pciutils.pm:24]: I would add the same also for libpci - wdyt?

Comment (Vogtinator) [reply tests/console/pciutils.pm:24]: libpci is a subpackage of pciutils - same version

**Review by Vogtinator — APPROVED** [review]

File: tests/console/pciutils.pm (hunk 2 above)
Comment (foursixnine) [reply tests/console/pciutils.pm:24]: Ah! fine then

**Review by foursixnine — APPROVED** [review]

**Pull request merged**